src/components/SamplePanel/index.js

```diff
@@ -140,15 +148,29 @@ const SamplePanel = ({sampleName, sampleData, config, reference, socket, panelEx
         }
         />
     )*/
+    mutationsTree: (
+      <MutationsTree
+       data={sampleVariant}
+       renderProp={ showSinglePanel === "mutationsTree" ?
+            (<ContractChart handleClick={() => goToChart(false)}/>) :
+            (<ExpandChart handleClick={() => goToChart("mutationsTree")}/>)
+        }
+      />
+    )
 
   };
 
   /* ---------------   WHAT CHARTS DO WE RENDER?   -------------- */
   const renderCharts = () => {
+  
     if (!panelExpanded) return null;
-    const chartsToShow = showSinglePanel ?
-    charts[showSinglePanel] :
-    [charts.coverage, charts.readLength, charts.coverageOverTime, charts.refSimilarity];
+    var chartsToShow =[];
+    if(variant_name!==""){
+      chartsToShow = showSinglePanel ? charts[showSinglePanel] : [charts.coverage, charts.readLength, charts.coverageOverTime, charts.refSimilarity, charts.mutationsTree];
+    }
+    else{
+      chartsToShow = showSinglePanel ? charts[showSinglePanel] : [charts.coverage, charts.readLength, charts.coverageOverTime, charts.refSimilarity];
```

Comment:
this feels a bit copy-paste. I guess it would better read
```
if (showSinglePanel) {
  chartsToShow = charts[...]
} else {
  chartsToShow = [coverage, ..., coverageOverTime]
  if (variant_name !== "") {
    chartsToShow.push(charts.refSimilarity)
  }
}
```